tests/pipelines/stable_cascade/test_stable_cascade_decoder.py

```diff
@@ -246,6 +247,32 @@ def test_stable_cascade_decoder_prompt_embeds(self):
 
         assert np.abs(decoder_output_prompt.images - decoder_output_prompt_embeds.images).max() < 1e-5
 
+    def test_stable_cascade_num_images_per_prompt(self):
```

Comment:
But we're not testing for `prompt_embeds_pooled` and `prompt_embeds` here. Shouldn't we test for those too?